shipyard_blueprints/s3/shipyard_s3/cli/authtest.py

```diff
@@ -1,31 +1,26 @@
 import os
 import sys
-import boto3
 from shipyard_s3 import S3Client
+from shipyard_templates import ShipyardLogger
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
```

Comment:
I understand the need for this running locally but we would probably want to remove this to be on the safe side when it goes to the application

Reply:
Yes this should be removed